gitlab/cli.py

```diff
@@ -151,7 +160,86 @@ def _get_base_parser(add_help: bool = True) -> argparse.ArgumentParser:
         ),
         required=False,
     )
+    parser.add_argument(
+        "--server-url",
+        help=("GitLab server URL [env var: GITLAB_URL]"),
+        required=False,
+        default=os.getenv("GITLAB_URL"),
+    )
+    parser.add_argument(
+        "--ssl-verify",
+        help=(
+            "Whether SSL certificates should be validated. [env var: GITLAB_SSL_VERIFY]"
```

Comment:
Are all of these environment variables documented somewhere besides here?

Reply:
They will get generated in the CLI reference documentation which I link to at the top of the CLI usage intro.

I agree it's not the nicest, but I also think the entire docs need a bit of a revamp to focus less on the configuration file (among other things).

I wasn't sure whether to add another manually maintained table that will start to go out of date :grin: just noticed the config file options are also incomplete. So maybe a single table with CLI/env/file config all in one would be best. With the current theme I think tables don't look so nice though, as the text doesn't really wrap. Should we open a follow-up?

Reply:
I'm fine with a follow-up. I just looked at the generated docs and I think it would be very easy to not find that info so a separate table to me seems best.

Reply:
Ok, I opened https://github.com/python-gitlab/python-gitlab/issues/1795 to track this.